build_runner/CHANGELOG.md

```diff
@@ -1,4 +1,21 @@
-## 0.7.13-prerelease
+## 0.8.0-prerelease
+
+### New Features
+
+- Added `applyPostProccess` method which takes `PostProcessBuilderFactory`s
+  instead of `BuilderFactory`s.
+
+### Breaking Changes
+
+- Split `BuildAction` into `BuilderBuildAction` and `PostProcessBuildAction`.
```

Comment:
Is this class public? If it isn't this isn't a breaking change. If it is we should probably figure out why